README.rst

```diff
@@ -14,10 +14,10 @@ Prerequisites
   1. ``Mesos`` Cluster (cf: `Mesos Getting Started <http://mesos.apache.org/documentation/latest/getting-started>`_). All nodes in the cluster should be reachable using their hostnames, and all nodes have identical ``/etc/passwd`` and ``/etc/group``.
   
   2. Setup ``Mesos Agent`` to enable `Mesos Containerizer <http://mesos.apache.org/documentation/container-image/>`_ and `Mesos Nvidia GPU Support <https://issues.apache.org/jira/browse/MESOS-4626>`_ (optional). eg:
-    ``mesos-agent --containerizers=mesos --image_providers=docker --isolation=filesystem/linux,docker/runtime,cgroups/devices,gpu/nvidia``
-    
+    ``mesos-agent --containerizers=mesos,docker --image_providers=docker --isolation=filesystem/linux,docker/runtime,cgroups/devices,gpu/nvidia``
```

Comment:
If I ran the command on a single node with master as following:
mesos-master sudo mesos-master --ip=xxx.xxx.xxx.xxx --work_dir=/MESOS_work_dir --log_dir=/MESOS_log_dir
and then started the mesos-agent as following:
sudo mesos-agent --advertise_ip=164.107.119.26 --master=mesos-master:5050 \
--log_dir=/MESOS_log_dir \
--work_dir=//MESOS_work_dir \
--containerizers=mesos --image_providers=docker \
--isolation=filesystem/linux,docker/runtime,cgroups/devices



WARNING: Logging before InitGoogleLogging() is written to STDERR
I1117 13:43:32.425282 15033 main.cpp:243] Build: 2016-11-15 11:08:13 by giorgio
I1117 13:43:32.425389 15033 main.cpp:244] Version: 1.0.1
I1117 13:43:32.427811 15033 logging.cpp:194] INFO level logging started!
I1117 13:43:32.431588 15033 systemd.cpp:237] systemd version `229` detected
I1117 13:43:32.431650 15033 main.cpp:342] Inializing systemd state
I1117 13:43:32.437474 15033 systemd.cpp:325] Started systemd slice `mesos_executors.slice`
I1117 13:43:32.439729 15033 containerizer.cpp:196] Using isolation: filesystem/linux,docker/runtime,cgroups/devices,network/cni
I1117 13:43:32.448791 15033 linux_launcher.cpp:101] Using /sys/fs/cgroup/freezer as the freezer hierarchy for the Linux launcher
E1117 13:43:32.460381 15033 shell.hpp:106] Command 'hadoop version 2>&1' failed; this is the output:
sh: 1: hadoop: not found
I1117 13:43:32.460695 15033 fetcher.cpp:62] Skipping URI fetcher plugin 'hadoop' as it could not be created: Failed to create HDFS client: Failed to execute 'hadoop version 2>&1'; the command was either not found or exited with a non-zero exit status: 127
I1117 13:43:32.477346 15033 main.cpp:434] Starting Mesos agent
I1117 13:43:32.478793 15042 slave.cpp:198] Agent started on 1)@164.107.119.26:5051
I1117 13:43:32.478826 15042 slave.cpp:199] Flags at startup: --advertise_ip="164.107.119.26" --appc_simple_discovery_uri_prefix="http://" --appc_store_dir="/tmp/mesos/store/appc" --authenticate_http_readonly="false" --authenticate_http_readwrite="false" --authenticatee="crammd5" --authentication_backoff_factor="1secs" --authorizer="local" --cgroups_cpu_enable_pids_and_tids_count="false" --cgroups_enable_cfs="false" --cgroups_hierarchy="/sys/fs/cgroup" --cgroups_limit_swap="false" --cgroups_root="mesos" --container_disk_watch_interval="15secs" --containerizers="mesos" --default_role="*" --disk_watch_interval="1mins" --docker="docker" --docker_kill_orphans="true" --docker_registry="https://registry-1.docker.io" --docker_remove_delay="6hrs" --docker_socket="/var/run/docker.sock" --docker_stop_timeout="0ns" --docker_store_dir="/tmp/mesos/store/docker" --docker_volume_checkpoint_dir="/var/run/mesos/isolators/docker/volume" --enforce_container_disk_quota="false" --executor_registration_timeout="1mins" --executor_shutdown_grace_period="5secs" --fetcher_cache_dir="/tmp/mesos/fetch" --fetcher_cache_size="2GB" --frameworks_home="" --gc_delay="1weeks" --gc_disk_headroom="0.1" --hadoop_home="" --help="false" --hostname_lookup="true" --http_authenticators="basic" --http_command_executor="false" --image_providers="docker" --image_provisioner_backend="copy" --initialize_driver_logging="true" --isolation="filesystem/linux,docker/runtime,cgroups/devices" --launcher_dir="/usr/local/libexec/mesos" --log_dir="//MESOS_log_dir" --logbufsecs="0" --logging_level="INFO" --master="mesos-master:5050" --oversubscribed_resources_interval="15secs" --perf_duration="10secs" --perf_interval="1mins" --port="5051" --qos_correction_interval_min="0ns" --quiet="false" --recover="reconnect" --recovery_timeout="15mins" --registration_backoff_factor="1secs" --revocable_cpu_low_priority="true" --sandbox_directory="/mnt/mesos/sandbox" --strict="true" --switch_user="true" --systemd_enable_support="true" --systemd_runtime_directory="/run/systemd/system" --version="false" --work_dir="/MESOS_work_dir"
I1117 13:43:32.480165 15042 slave.cpp:519] Agent resources: cpus(*):4; mem(*):14979; disk(*):213466; ports(*):[31000-32000]
I1117 13:43:32.480238 15042 slave.cpp:527] Agent attributes: [  ]
I1117 13:43:32.480264 15042 slave.cpp:532] Agent hostname: mst
I1117 13:43:32.485796 15040 state.cpp:57] Recovering state from '/MESOS_work_dir/meta'
I1117 13:43:32.487273 15038 status_update_manager.cpp:200] Recovering status update manager
I1117 13:43:32.487586 15040 containerizer.cpp:522] Recovering containerizer
I1117 13:43:32.492664 15039 metadata_manager.cpp:205] No images to load from disk. Docker provisioner image storage path '/tmp/mesos/store/docker/storedImages' does not exist
I1117 13:43:32.493129 15039 provisioner.cpp:253] Provisioner recovery complete
I1117 13:43:32.493494 15038 slave.cpp:4782] Finished recovery
I1117 13:43:32.493809 15038 slave.cpp:4815] Garbage collecting old agent 49e17081-7c0c-4666-8e89-cb156774d71f-S15
I1117 13:43:32.493922 15042 gc.cpp:55] Scheduling '/MESOS_work_dir/slaves/49e17081-7c0c-4666-8e89-cb156774d71f-S15' for gc 6.99999428408593days in the future
I1117 13:43:32.493948 15038 slave.cpp:4815] Garbage collecting old agent 77f18644-a804-4040-9bd7-eec82fd42263-S0
I1117 13:43:32.494364 15042 gc.cpp:55] Scheduling '/MESOS_work_dir/meta/slaves/49e17081-7c0c-4666-8e89-cb156774d71f-S15' for gc 6.99999428329778days in the future
I1117 13:43:32.494634 15039 status_update_manager.cpp:174] Pausing sending status updates
I1117 13:43:32.494653 15038 slave.cpp:895] New master detected at master@164.107.119.26:5050
I1117 13:43:32.494774 15038 slave.cpp:916] No credentials provided. Attempting to register without authentication
I1117 13:43:32.494807 15042 gc.cpp:55] Scheduling '/MESOS_work_dir/slaves/77f18644-a804-4040-9bd7-eec82fd42263-S0' for gc 6.99999428217482days in the future
I1117 13:43:32.494910 15038 slave.cpp:927] Detecting new master
I1117 13:43:32.494945 15042 gc.cpp:55] Scheduling '/MESOS_work_dir/meta/slaves/77f18644-a804-4040-9bd7-eec82fd42263-S0' for gc 6.99999428163259days in the future
I1117 13:43:33.277678 15040 slave.cpp:809] Agent asked to shut down by master@164.107.119.26:5050 because 'Agent attempted to re-register after removal'
I1117 13:43:33.277853 15040 slave.cpp:767] Agent terminating